packages/fractal-page-object/API.md

```diff
@@ -93,21 +94,18 @@ class Page extends PageObject {
 
 setRoot(rootElement);
 
-let page = new Page();
-page.list.elements; // rootElement.querySelectorAll('.list')
-```
-
-```javascript
-import { PageObject, selector, setRoot } from 'fractal-page-object';
-
-class Page extends PageObject {
-  list = selector('.list');
-}
-
-setRoot(rootElement);
-
-let page = new Page('.container');
-page.list.elements; // rootElement.querySelectorAll('.container .list')
+// rootElement.querySelectorAll('.list')
+new Page().list.elements;
+// rootElement.querySelectorAll('.container .list')
+new Page('.container').list.elements;
+// rootElement.querySelectorAll('.container')[0].querySelectorAll('.list')
+new Page('.container', null, 0).list.elements;
+// document.body.querySelectorAll('.list');
+new Page('', document.body).list.elements;
+// document.body.querySelectorAll('.container .list');
```

Comment:
Isn't this something like?
```
// document.body.querySelectorAll('.container .list').elements
```

Reply:
`document.body.querySelectorAll('.container .list')` returns a `NodeListOf<Element>`, which is basically an iterable of elements. `PageObject(...).elements` returns `Element[]`, which is the equivalent in the mental model here. The page object is an object that describes a sorta-location in the DOM (like a selector does), and accessing `elements` gives you the actual matching elements like `querySelectorAll()` does.

Does that make sense? Or did I misunderstand your question?

Reply:
Ah yes, thanks for clarifying (I'm so used to the PageObject API I forgot what querySelectorAll does).